requirements-docs.txt

```diff
@@ -6,6 +6,7 @@ numpydoc
 pybtex
 pygithub
 sphinx>=4.3
-sphinx-panels
+sphinx-autodoc-typehints>=2.0
+sphinx-design
```

Comment:
I wasn't able to upgrade to get the needed docstring parsing without dropping `sphinx-panels`. I could probably figure out how with enough time but the migration was simple enough.

Reply:
Ah - I was wondering what motivated this and GH didn't give me this review comment over in the diff view. 🤦‍♀️

Reply:
So many little annoying things like that with GitHub PRs that I expected to be fixed by now!